executor/analyze.go

```diff
@@ -40,9 +37,9 @@ type AnalyzeExec struct {
 }
 
 const (
-	maxSampleCount     = 10000
-	maxSketchSize      = 1000
-	defaultBucketCount = 256
+	maxSampleSize = 10000
+	maxSketchSize = 1000
```

Comment:
How about keep the same sample size 10000 and move this initialization to stats package ?

Reply:
I think it is better to keep it here because it is used to control the behavior of stats, put it in the caller part seems more reasonable.

Reply:
How about keeping maxSketchSize 10000 ?

Reply:
Done.